app/src/lib/editors/shared.ts

```diff
@@ -16,3 +18,16 @@ export type FoundEditor = {
 export const AtomLabel = 'Atom'
 export const VisualStudioCodeLabel = 'Visual Studio Code'
 export const SublimeTextLabel = 'Sublime Text'
+
+/**
+ * Helper function to promisify fs.exists
+ *
+ * @param path Path to check for existence.
+ */
+export function pathExists(path: string): Promise<boolean> {
+  return new Promise<boolean>((resolve, reject) => {
+    Fs.exists(path, exists => {
```

Comment:
`fs.exists` is deprecated https://nodejs.org/api/fs.html#fs_fs_exists_path_callback